tests/cross_functional/system_test/test_cluster_wide_key_rotation_systemtest.py

```
@@ -185,6 +194,6 @@ def verify_key_rotation_time(self, schedule):
         try:
             validate_key_rotation_schedules(schedule=schedule)
         except ValueError as e:
-            log.error(f"Validation failed: {e}")
+            logger.error(f"Validation failed: {e}")
```

Ruff (0.15.21)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)